activitypub-core/src/main/java/org/xwiki/contrib/activitypub/entities/ActivityPubObject.java

```diff
@@ -387,6 +388,7 @@ public String getXwikiReference()
      * @since 1.4
      */
     @Unstable
+    @JsonDeserialize(using = ActivityPubObjectReferenceArrayJsonDeserializer.class)
```

Comment:
I don't understand this: thought we agreed on having the same behaviour for any property, not just tag.